### PR TITLE
Feature/object detection

### PR DESCRIPTION
**Pull request by Ayush-sharma-py**

### CODE REVIEW

File: src/perception/object_detection_2D/test/test_pep257.py
Comment (danielv012) [inline src/perception/object_detection_2D/test/test_pep257.py:1]: What's this?

File: src/perception/object_detection_2D/test/test_flake8.py
Comment (danielv012) [inline src/perception/object_detection_2D/test/test_flake8.py:1]: What's this?

File: src/perception/object_detection_2D/test/test_copyright.py
Comment (danielv012) [inline src/perception/object_detection_2D/test/test_copyright.py:1]: What's this?

File: my_package/test/test_pep257.py
Comment (danielv012) [inline my_package/test/test_pep257.py:1]: delete

File: my_package/test/test_flake8.py
Comment (danielv012) [inline my_package/test/test_flake8.py:1]: delete

File: my_package/test/test_copyright.py
Comment (danielv012) [inline my_package/test/test_copyright.py:1]: delete

File: my_package/setup.py
Comment (danielv012) [inline my_package/setup.py:1]: delete

File: my_package/setup.cfg
Comment (danielv012) [inline my_package/setup.cfg:1]: delete

File: my_package/resource/my_package
Comment (danielv012) [inline my_package/resource/my_package:1]: delete whole directory

File: my_package/package.xml
Comment (danielv012) [inline my_package/package.xml:1]: delete

File: my_package/my_package/my_node.py
Comment (danielv012) [inline my_package/my_package/my_node.py:1]: delete

File: my_package/my_package/__init__.py
Comment (danielv012) [inline my_package/my_package/__init__.py:1]: delete

File: launches/launch.vehicle.py [hunk 1]
```diff
@@ -52,6 +52,13 @@ def generate_launch_description():
                     # PERCEPTION
                     *perception_launch_entities,
 
+                    # image_segmentation,
+                    # semantic_projection,
+                    #carla_lidar_processor,
+
+                    # prednet_inference,
+                    # driveable_area,
+
```
Comment (danielv012) [inline launches/launch.vehicle.py:61]: unnecessary changes

File: docker/entrypoint.sh [hunk 2]
```diff
@@ -35,9 +35,10 @@ echo "🔧 Setting up aliases..."
 echo "alias navigator=\"python3 /navigator/navigator.py\"" >> ~/.bashrc
 
 echo "👍 Finished environment setup"
-
+#export PYTHONPATH=$PYTHONPATH:"/models/research":"/models/research/slim"
```
Comment (danielv012) [inline docker/entrypoint.sh:38]: remove

File: Dockerfile [hunk 3]
```diff
@@ -220,7 +220,56 @@ RUN usermod -a -G tty root
 
 ENV ROS_VERSION 2
 
+
+
+# TESTING
+# Use Ubuntu as the base image
+#FROM ubuntu:latest
```
Comment (danielv012) [inline Dockerfile:227]: can remove

File: Dockerfile [hunk 4]
```diff
@@ -220,7 +220,56 @@ RUN usermod -a -G tty root
...
+    protoc object_detection/protos/*.proto --python_out=. 
+    #&& \
+    #pip3 install .
+RUN apt-get install ffmpeg libsm6 libxext6  -y
+
+# Adding files used locally
+#ADD src/perception/object_detection/test.sh models/
+#ADD src/perception/object_detection/detection.py models/research/object_detection/
+#ADD src/perception/object_detection/objDec.py models/research/object_detection/
+#ADD src/perception/object_detection/cabc30fc-e7726578.mov models/research/object_detection/
```
Comment (danielv012) [inline Dockerfile:267]: Can remove

File: Dockerfile [hunk 5]
```diff
@@ -220,7 +220,56 @@ RUN usermod -a -G tty root
...
+
+
+# TESTING
```
Comment (danielv012) [inline Dockerfile:270]: can remove

**Review by danielv012 — CHANGES_REQUESTED** [review]
Everything looks great, just need to trim a few things

**Comment by Ayush-sharma-py** [comment]
Making a new Pull Request with the changes

**Pull request closed without merging**